machine/completion.md

```diff
@@ -14,12 +14,9 @@ for the bash and zsh shell.
 Make sure bash completion is installed. If you use a current Linux in a non-minimal installation, bash completion should be available.
 On a Mac, install with `brew install bash-completion`
 
-Place the completion scripts in `/etc/bash_completion.d/` (`` `brew --prefix`/etc/bash_completion.d/`` on a Mac), using e.g.
+Place the completion script in `/etc/bash_completion.d/` (`` remove `brew --prefix`/etc/bash_completion.d/`` on a Linux), using e.g.
```

Comment:
Oh, yes the "on a Linux" part is incorrect. I just replaced "Mac" with "Linux" in this line, thats why the error is in there. The remove part states, that you have to remove the "brew --prefix`/etc/bash_completion.d/" from the following command if you use Linux.